src/main/resources/anniv5Resources/localization/eng/metapack/Cardstrings.json

```diff
@@ -0,0 +1,46 @@
+{
+  "${ModID}:ShedWeight": {
+    "NAME": "Shed Weight",
+    "DESCRIPTION": "Gain !B! Block. NL Discard a card. NL If that card was *Unplayable, draw a card."
+  },
+  "${ModID}:RayOfFrost": {
+    "NAME": "Ray Of Frost",
+    "DESCRIPTION": "*Magic. NL Deal !D! damage. NL Whenever you play a *Magic card this turn, Channel !M! Frost."
+  },
+  "${ModID}:Swipe": {
+    "NAME": "Swipe",
+    "DESCRIPTION": "Deal !D! damage for each point of Energy missing.",
+    "EXTENDED_DESCRIPTION": [" NL (Hits !M! times.)"]
+  },
+  "${ModID}:Ambition": {
+    "NAME": "Ambition",
+    "DESCRIPTION": "Put a random Power from your draw pile into your hand and copy it. NL Exhaust.",
+    "UPGRADE_DESCRIPTION": "Innate. NL Put a random Power from your draw pile into your hand and copy it. NL Exhaust."
+  },
+  "${ModID}:Diehard": {
+    "NAME": "Diehard",
+    "DESCRIPTION": "Gain !M! Artifact. NL Next turn, gain 1 Weak and Frail."
+  },
+  "${ModID}:EldritchBlast": {
+    "NAME": "Eldritch Blast",
+    "DESCRIPTION": "*Magic. NL Deal !D! damage. Reduce the cost of a random *Magic card in hand by 1 this turn."
+  },
+  "${ModID}:DeathWish": {
+    "NAME": "Death Wish",
+    "DESCRIPTION": "If your HP is above 50%, lose !M! HP. NL Gain !M! Strength and *Vigor."
+  },
+  "${ModID}:KillerInstinct": {
+    "NAME": "Killer Instinct",
+    "DESCRIPTION": "Gain 2 Strength and 1 Dexterity. NL Startup: Shuffle a copy of this card into your draw pile.",
+    "UPGRADE_DESCRIPTION": "Gain 2 Strength and 1 Dexterity. NL Startup: Shuffle !M! copies of this card into your draw pile."
+  },
+  "${ModID}:Bulwark": {
+    "NAME": "Bulwark",
+    "DESCRIPTION": "Innate. NL Gain Temporary HP equal to your draw pile size. NL Exhaust.",
```

Comment:
```suggestion
    "DESCRIPTION": "Innate. NL Gain Temporary HP equal to the amount of cards in your draw pile. NL Exhaust.",
```

Reply:
This bumps the description to 6 lines. Wording is the same as Magic Dart, it should be instantly comprehensible.